contrib/integration_tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+KPATCH_SLOW=0
+LOGDIR="/vagrant/logs"
+
+usage()
+{
+  echo "usage: $(basename "${0}") [options]" >&2
+  echo "-h, --help This message" >&2
+  echo "-s, --slow Run all of the tests" >&2
+}
+
+options="$(getopt -o s -l "slow" -- "$@")" || "getopt failed"
+
+eval set -- "${options}"
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    -s|--slow)
+      KPATCH_SLOW=1
+      ;;
+    -h|--help)
+      usage
+      exit 0
+      ;;
+  esac
+  shift
+done
+
+git clone https://github.com/dynup/kpatch.git || exit 1
+
+cd kpatch || exit 1
+
+# shellcheck disable=SC1091
+source contrib/lib.sh
```

Comment:
Uh, that's because we clone the official repo inside the vm and this PR is no merged yet, if you want it to run you'll need to change line 30 in contrib/integration_tests.sh to `git clone -b intpud https://github.com/sm00th/kpatch`

Reply:
Ha, of course...